### DISCUSSION TITLE
AbstractClientEntity: isUpdated apparently missing

### DISCUSSION THREAD

**Discussion by sguilhen** [keycloak/keycloak#10921] [Archive]
I've noticed that the [AbstractClientEntity](https://github.com/keycloak/keycloak/blob/main/model/map/src/main/java/org/keycloak/models/map/client/MapClientEntity.java#L41) is not implementing the `isUpdate` method to account for changes in the `MapProtocolMapperEntity` set. Other abstract entities, like the [AbstractRealmEntity](https://github.com/keycloak/keycloak/blob/main/model/map/src/main/java/org/keycloak/models/map/realm/MapRealmEntity.java#L69), do override the method to account for changes in the associated entities.

Shouldn't the `AbstractClientEntity` implement it too or am I missing something?

**Comment by hmlnarik** [comment]
I believe it should, thanks for noticing it!

**Reply by sguilhen** [reply]
Thanks, I've created #10927
